src/vision/customVisionTraining.js

```diff
@@ -0,0 +1,210 @@
+const commonService = require('../commonService');
+
+/**
+Custom Vision Service brings the power of machine learning to your apps
+Custom Vision Service is a tool for building custom image classifiers. 
+It makes it easy and fast to build, deploy, and improve an image classifier. 
+We provide a REST API and a web interface to upload your images and train.
+
+Prerequisites
+-To build a classifier, you must first have:
+-A valid Microsoft Account or an Azure Active Directory OrgID ("work or school account"), so you can sign into customvision.ai and get started. Note that OrgID login for AAD users from national clouds is not currently supported.
+-A series of images to train your classifier (minimum of 30 images per tag).
+-A few images to test your classifier after the classifier is trained.
+ * @augments commonService
+ * {@link https://westus.dev.cognitive.microsoft.com/docs/services/56f91f2d778daf23d8ec6739/operations/56f91f2e778daf14a499e1fa|documentation}
```

Comment:
Correct link is https://southcentralus.dev.cognitive.microsoft.com/docs/services/d9a10a4a5f8549599f1ecafc435119fa/operations/58d5835bc8cb231380095be3